db/migrate/20160304185304_disable_sms_authentication_on_forms.rb

```diff
@@ -0,0 +1,13 @@
+class DisableSmsAuthenticationOnForms < ActiveRecord::Migration
+  def up
+    Form.find_each do |f|
```

Comment:
I was not aware of this method. Cool.

But wouldn't `update_all` be more efficient here?